File: BackToSenderLogistics.java
import java.util.Scanner;

public class BackToSenderLogistics{

	public static void payment(int... number){

	Scanner input = new Scanner(System.in);
	int basePay = 5000;

	System.out.print("How many successful delivery was made: ");
	int successfulDelivery = input.nextInt();
	
	int amountPerPacel = 0;

	if(successfulDelivery < 50){
	amountPerPacel = 160;
	}

	else if(successfulDelivery >= 50 && successfulDelivery <= 59){

	amountPerPacel = 200;

	}

	else if(successfulDelivery >= 60 && successfulDelivery <= 69){

	amountPerPacel = 250;

	}

	else{

	amountPerPacel = 500;

	}

	int totalPay = successfulDelivery * amountPerPacel + basePay;

	System.out.print("Total ppayment for the day is " + totalPay);

}

	public static void main(String... args){

	payment();
}

}

File: back_to_sender_logistics.py
def back_to_sender_logistics():

	base_pay = 5000;

	successful_deliveries = int(input("Enter successful deliveries: "))

	rate = 0

	if successful_deliveries < 50:

		rate = 160

	elif successful_deliveries >= 50 and successful_deliveries <= 59:

		rate = 250

	elif successful_deliveries >= 60 and successful_deliveries <= 69:

		rate = 400

	elif successful_deliveries >= 70:

		rate = 500

	total_wage = (successful_deliveries * rate) + base_pay

	return total_wage

print("Wage for the day is:",back_to_sender_logistics())


		